posthog/api/test/test_organization.py

```diff
@@ -69,17 +69,22 @@ def test_cant_create_organization_without_valid_license_on_self_hosted(self):
 
     # Updating organizations
 
-    def test_update_organization_if_admin(self):
-        self.organization_membership.level = OrganizationMembership.Level.ADMIN
-        self.organization_membership.save()
-        response_rename = self.client.patch(f"/api/organizations/{self.organization.id}", {"name": "QWERTY"})
-        response_email = self.client.patch(
-            f"/api/organizations/{self.organization.id}", {"is_member_join_email_enabled": False}
-        )
-        self.assertEqual(response_rename.status_code, status.HTTP_200_OK)
-        self.assertEqual(response_email.status_code, status.HTTP_200_OK)
-        self.organization.refresh_from_db()
-        self.assertEqual(self.organization.name, "QWERTY")
+    def test_update_organization_if_admin_or_owner(self):
+        for level in (OrganizationMembership.Level.ADMIN, OrganizationMembership.Level.OWNER):
+            self.organization_membership.level = level
+            self.organization_membership.save()
+            response_rename = self.client.patch(f"/api/organizations/{self.organization.id}", {"name": "QWERTY"})
+            response_email = self.client.patch(
+                f"/api/organizations/{self.organization.id}", {"is_member_join_email_enabled": False}
+            )
+            response_per_project_access = self.client.patch(
+                f"/api/organizations/{self.organization.id}", {"per_project_access": False}
+            )
+            self.assertEqual(response_rename.status_code, status.HTTP_200_OK)
+            self.assertEqual(response_email.status_code, status.HTTP_200_OK)
+            self.assertEqual(response_per_project_access.status_code, status.HTTP_200_OK)
+            self.organization.refresh_from_db()
+            self.assertEqual(self.organization.name, "QWERTY")
```

Comment:
Check `is_member_join_email_enabled` & `per_project_access` in the model are updated too.